content/basics/objects.md

```diff
@@ -78,15 +75,9 @@ you to quickly and easily select elements in an array. eg.
 
 ```js
 // spread
-const list = [10, 12, 13];
-const [ ..., a, b ] = list;
```

Comment:
Invalid syntax 😄  `rest` must be the last parameter. 

Perhaps there should also be a section for object spread too?